pkg/piecestore/pstore.go

```diff
@@ -66,7 +67,7 @@ func Store(hash string, r io.Reader, length int64, psFileOffset int64, dir strin
 		return 0, err
 	}
 
-	dataFileChunk := fpiece.NewChunk(dataFile, psFileOffset, length)
+	dataFileChunk, _ := fpiece.NewChunk(dataFile, psFileOffset, length)
```

Comment:
@aleitner @kaloyan-raev 

hey! check all your errors! tsk tsk

@coyle isn't errcheck part of the linting?

Reply:
Alex asked in the PR description to ignore this file, because it will be changed by the upcoming PR #40.

So did I.